build.gradle

```diff
@@ -36,6 +47,51 @@ dependencies {
   mlcp files("mlcp/lib")
 }
 
+/**
+ * START: Disconnected gradle tasks
+ * Reference: https://github.com/derms/ml-gradle/blob/dev/examples/disconnected-project-using-plugins-and-gradlew/build.gradle
```

Comment:
it was merged into ml-com: https://github.com/marklogic-community/ml-gradle/tree/master/examples/disconnected-project-using-plugins-and-gradlew